libcontainerd/client_local_windows.go

```diff
@@ -585,6 +564,36 @@ func (c *client) createLinux(id string, spec *specs.Spec, runtimeOptions interfa
 	return nil
 }
 
+func (c *client) setWindowsCPUMemoryLimits(spec *specs.Spec, configuration *hcsshim.ContainerConfig) {
```

Comment:
Nit - Might be more obvious if this was called something more like `extractResourcesFromSpec`